<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>320</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>460</width>
    <height>320</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>36</pointsize>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>CrossCursor</cursorShape>
  </property>
  <property name="tabletTracking">
   <bool>false</bool>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>461</width>
      <height>271</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(22, 22, 57);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Lights">
     <attribute name="title">
      <string>Lights</string>
     </attribute>
     <widget class="QPushButton" name="On">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>40</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(9, 5, 3);</string>
      </property>
      <property name="text">
       <string>On</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Scene">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>130</y>
        <width>121</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(4, 1, 1);</string>
      </property>
      <property name="text">
       <string>Scene</string>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="Off">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>40</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(9, 5, 3);</string>
      </property>
      <property name="text">
       <string>Off</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Red">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>130</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(4, 1, 1);</string>
      </property>
      <property name="text">
       <string>Red</string>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="Colour">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>130</y>
        <width>111</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(4, 1, 1);</string>
      </property>
      <property name="text">
       <string>Color</string>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="White">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(4, 1, 1);</string>
      </property>
      <property name="text">
       <string>White</string>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="MusicTab">
     <attribute name="title">
      <string>Music</string>
     </attribute>
     <widget class="QToolBox" name="toolBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>411</width>
        <height>211</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Controls">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>411</width>
         <height>131</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Controls</string>
       </attribute>
       <widget class="QPushButton" name="btnOnplay">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>0</y>
          <width>162</width>
          <height>57</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>36</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>PLAY</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnOffpause">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>60</y>
          <width>162</width>
          <height>57</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>36</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>PAUSE</string>
        </property>
       </widget>
       <widget class="QPushButton" name="prevsong">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>30</y>
          <width>111</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(43, 43, 43, 194);
color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Prev. Song</string>
        </property>
       </widget>
       <widget class="QPushButton" name="nextsong">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>30</y>
          <width>111</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(43, 43, 43, 194);
color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Next Song</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>60</y>
          <width>81</width>
          <height>81</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(55, 125, 46);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../../Downloads/file-spotify-logo-png-4_2_150x150_1_50.png</pixmap>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Volume">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>100</width>
         <height>30</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Volume</string>
       </attribute>
       <widget class="QPushButton" name="Mute">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>30</y>
          <width>171</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(43, 43, 43, 194);
color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Mute</string>
        </property>
       </widget>
       <widget class="QPushButton" name="VolDown">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>60</y>
          <width>131</width>
          <height>71</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(43, 43, 43, 194);
color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Vol. Down</string>
        </property>
       </widget>
       <widget class="QPushButton" name="VolUp">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>0</y>
          <width>131</width>
          <height>71</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(43, 43, 43, 194);
color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Vol. Up</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="inputs">
     <attribute name="title">
      <string>Inputs</string>
     </attribute>
     <widget class="QToolBox" name="toolBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>421</width>
        <height>211</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="video">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>421</width>
         <height>131</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Video</string>
       </attribute>
       <widget class="QPushButton" name="DP">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>0</y>
          <width>161</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>48</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(4, 1, 1);</string>
        </property>
        <property name="text">
         <string>DP</string>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="HDMI">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>0</y>
          <width>161</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>48</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(9, 5, 3);</string>
        </property>
        <property name="text">
         <string>HDMI</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_10">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>421</width>
         <height>131</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Audio</string>
       </attribute>
       <widget class="QPushButton" name="AirPods">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>241</width>
          <height>57</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>36</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(4, 1, 1);</string>
        </property>
        <property name="text">
         <string>AirPods</string>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="FOSI">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>60</y>
          <width>241</width>
          <height>57</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>36</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(4, 1, 1);</string>
        </property>
        <property name="text">
         <string>FOSI</string>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="btoff">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>0</y>
          <width>141</width>
          <height>57</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>36</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(9, 5, 3);</string>
        </property>
        <property name="text">
         <string>BT Off</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action2_0">
   <property name="text">
    <string>2.0</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
